src/ILCompiler.WebAssembly/src/CodeGen/WebAssemblyObjectWriter.cs

```diff
@@ -187,6 +188,28 @@ public void FinishObjWriter()
             //throw new NotImplementedException(); // This function isn't complete
         }
 
+        public static LLVMValueRef EmitGlobal(LLVMModuleRef module, FieldDesc field, NameMangler nameMangler)
```

Comment:
@davidwrighton can provide more detail on this, but I suspect this needs to be driven by the ObjectNode infrastructure